src/functions/vacation-request/create-vacation-request/handler.ts

```diff
@@ -18,7 +18,19 @@ export const createVacationRequestHandler: ValidatedEventAPIGatewayProxyEvent<ty
     };
   }
 
-  const { userId, draft, startDate, endDate, days, type, status, message, createdBy, createdAt, updatedAt} = event.body;
+  const { body: {
+    createdAt,
+    createdBy,
+    days,
+    draft,
+    endDate,
+    message,
+    startDate,
+    status,
+    type,
+    updatedAt,
+    userId
+  }} = event;
 
```

Comment:
Could this be:
`const { body } = event
const {
    createdAt,
    createdBy,
    days,
    draft,
    endDate,
    message,
    startDate,
    status,
    type,
    updatedAt,
    userId
} = body;`

Like in some of the other projects? just a thought.